togglReader.go

```diff
@@ -105,8 +105,10 @@ func (w TogglWrapper) CurrentTimer() Timer {
 	_, err = jsonparser.ArrayEach(requestString, func(value []byte, dataType jsonparser.ValueType, offset int, err error) {
 		tags = append(tags, string(value))
 	}, "data", "tags")
-	if err != nil {
+	if err != nil && err != jsonparser.KeyPathNotFoundError {
 		log.Fatalf("Error iterating through tags array in JSON of current timer: %s", err)
+	} else if err == jsonparser.KeyPathNotFoundError {
+		tags = append(tags,"")
```

Comment:
Just assign `nil` if we reach the KeyPathNotFoundError:

```suggestion
	} else if err == jsonparser.KeyPathNotFoundError {
		tags = nil
```

But you'll also need to modify the tags variable creation so that `tags` is originally a nil slice, like:
```diff
@@ -101,7 +101,7 @@ func (w TogglWrapper) CurrentTimer() Timer {
 		log.Fatalf("Error parsing time string from time entry: %s", err)
 	}
 
-	tags := make([]string, 0)
+	var tags []string
 	_, err = jsonparser.ArrayEach(requestString, func(value []byte, dataType jsonparser.ValueType, offset int, err error) {
 		tags = append(tags, string(value))
 	}, "data", "tags")

```

Reply:
Looks like this introduces another error on line 51, when you randomly pick the icon to display:
```go
iconId := timeEntry.tags[rand.Intn(len(timeEntry.tags))]
```
results in:
```
panic: invalid argument to Intn
```
I think we would also need to add some kind of check to make sure tags is not nil.